OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -130,10 +130,6 @@ void INotifyCreated.Created(Actor self)
 			notifyHarvesterAction = self.TraitsImplementing<INotifyHarvesterAction>().ToArray();
 			conditionManager = self.TraitOrDefault<ConditionManager>();
 			UpdateCondition(self);
-
-			// Note: This is queued in a FrameEndTask because otherwise the activity is dropped/overridden while moving out of a factory.
-			if (Info.SearchOnCreation)
```

Comment:
`Info.SearchOnCreation` is now unused.  This doesn't seem right, because there should be a way to make harvesters *not* automatically look for resources when built.

Reply:
The idea was to have only one place from which harvester activities are called aside from explicit user orders. I've added a conditional to fix the behaviour.